lib/index.js

```diff
@@ -1,3 +1,7 @@
-var TwitterOauth2Strategy = require("./oauth2");
+var Strategy = require("./oauth2");
 
-module.exports = TwitterOauth2Strategy;
+// Expose Strategy.
+exports = module.exports = Strategy;
+
+// Exports.
+exports.Strategy = Strategy;
```

Comment:
since i changed from passport-twitter to this library, it was not working as i expected, so i changed it to work similar to that other library

`
import { Strategy, ... } from "passport-twitter-oauth2.0";
@Injectable()
export class TwitterStrategy extends PassportStrategy(Strategy, "twitter") {
    constructor(
        private readonly configService: ConfigService,
        private readonly userService: UserService,
        private readonly socialProfileService: SocialProfileService,
    ) {
        super({
            clientID: configService.getTwitterOAuth2ClientID(),
            clientSecret: configService.getTwitterOAuth2ClientSecret(),
            callbackURL: configService.getTwitterCallbackUrl(),
            clientType: "private",
            pkce: true,
            state: true,
            scope: ["offline.access", "users.read", "tweet.read", "follows.read", "like.read"],
        });
    }
}
`